### PR TITLE
Add signal analyzer and gps registration signals. 

### PR DESCRIPTION
**Pull request by dboulware**
This adds signals to register signal analyzer and gps.

### CODE REVIEW

File: scos_actions/signals.py [hunk 1]
```diff
@@ -13,3 +13,9 @@
 
 # Provides argument 'action'
 register_action = Signal()
+
+# Provides argument 'signal_analyzer'
+register_signal_analyzer = Signal()
```
Comment (aromanielloNTIA) [inline scos_actions/signals.py:18]: I think this PR is missing the registrars/monitors (which were previously included in #107)

Comment (aromanielloNTIA) [reply scos_actions/signals.py:18]: These^ are now going to be included in scos-sensor instead

**Review by aromanielloNTIA — APPROVED** [review]
Looks good!

**Comment by dboulware** [comment]
Working on a a better approach to fix a larger architectural  issue.

**Pull request closed without merging**